pkg/commands/user.go

```diff
@@ -56,8 +57,18 @@ func (r *UserCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.Bu
 	}
 
 	userStr = userStr + ":" + groupStr
-
 	config.User = userStr
+
+	// Remove any HOME environment variable previously set.
```

Comment:
What happens if `USER` is the last command in a dockerfile.

```
FROM image as a
ENV HOME /usr/home
USER test:test
```

Reply:
What would docker do?

Reply:
Like you think it would, `ENV HOME` prevails. Thanks!